content/docs/start/experiments.md

```diff
@@ -18,17 +18,87 @@ once they're no longer needed.
 
 ## Running experiments
 
-Previously, we learned how to tune [ML pipelines](/doc/start/data-pipelines) and
-[compare the changes](/doc/start/metrics-parameters-plots). Let's further
-increase the number of features in the `featurize` stage to see how it compares.
+In the previous page, we learned how to tune
+[ML pipelines](/doc/start/data-pipelines) and compare the changes. In this
+section we'll use a
+[Get Started to Experiments project](https://github.com/iterative/get-started-experiments)
+to illustrate experimentation features in DVC 2.0
+
+<details>
+
+### 🆕 Click here for the instructions to install the project
+
+These commands are run in
+[`get-started-experiments`](https://github.com/iterative/get-started-experiments)
+project. You can run these commands after cloning the repository and install the
+requirements.
+
+Please clone and create a virtual environment:
+
+```console
+git clone https://github.com/iterative/get-started-experiments
+cd get-started-experiments
+python -m venv .venv
+. .venv/bin/activate
+python -m pip install -r requirements.txt
+```
+
+Then you can `dvc pull` to get the dataset and run the commands in this
+document. For detailed information on parameters and the project structure
+please refer to the
+[project repository](https://github.com/iterative/get-started-experiments)
+
+</details>
 
 `dvc exp run` makes it easy to change <abbr>hyperparameters</abbr> and run a new
-experiment:
+experiment. We'll use it to search for parameters to increase the classification
+performance of our model using
+[Fashion-MNIST](https://github.com/iterative/get-started-experiments) dataset.
+
+In order to run a baseline experiment with the default parameters defined in
+`params.yaml`:
+
+```dvc
+dvc exp run
+```
+
+![dvc exp run result screenshot](/img/doc/start/exp-ss-90252.png)
```

Comment:
here screenshot is probably not needed?

Reply:
I had thought something about putting the text instead but decided to put the screenshot but the reason seems not something worthwhile to remember. 😅  I'm removing it now.